proto/errors.proto

```diff
@@ -192,6 +193,11 @@ enum TransactionRestart {
   IMMEDIATE = 2;
 }
 
+// ErrPosition describes the position of an error in a Batch.
+message ErrPosition {
```

Comment:
yes, in proto3 primitive types are not nullable. there was a discussion on this in a previous PR